apex_launchtest/apex_launchtest/io_handler.py

```diff
@@ -45,9 +47,11 @@ def processes(self):
 
         :returns [launch.actions.ExecuteProcess]:
         """
-        return map(
-            lambda x: x.action,
-            [self._process_name_dict[name][0] for name in self._process_name_dict]
+        return list(
+            map(
+                lambda x: x.action,
+                [self._process_name_dict[name][0] for name in self._process_name_dict]
```

Comment:
What's the reason for the `map()`? Couldn't you just:

```python
return [processes[0].action for processes in self._process_name_dict.values()]
```

Also, the `[some_dict[name]... for name in some_dict]` is a bit weird, as opposed to `[value... for value in some_dict.values()]`.

Reply:
Yeah, you're right.  This is kind of a logic puzzle right now.